docs/GITOPS-CONFIG.md

```diff
@@ -1,69 +1,13 @@
 ## GitOps/ArgoCD Configuration
 
-As part of this section you will find information about the following configuration methods:
+1. [AI-RHDH-Installer Provisioned Operator Configuration](#ai-rhdh-installer-provisioned-operator-configuration)
+2. [Pre-Existing Operator Configuration](#pre-existing-operator-configuration)
```

Comment:
I think we can remove these redirects, its a very small document and the page doesnt even move much, it will just add more to the confusion